script.reddit.reader/addon.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon
+    id="script.reddit.reader"
+    name="Reddit Reader"
+    version="0.5.7"
+    provider-name="gedisony">
+    <requires>
+        <import addon="xbmc.python"                     version="2.14.0"                />
+        <import addon="xbmc.addon"                      version="13.0.0"                />
+        <import addon="plugin.video.youtube"            version="5.0.0"                 />
+        <import addon="plugin.video.vimeo"              version="4.0.0" optional="true" />
+        <import addon="plugin.video.dailymotion_com"    version="1.0.0" optional="true" />
```

Comment:
Why do you require all these addons? Shouldnt script.module.youtube.dl be able to deal with most of it?